app/helpers/application_helper/button/miq_ae.rb

```diff
@@ -0,0 +1,37 @@
+class ApplicationHelper::Button::MiqAe < ApplicationHelper::Button::Basic
+  include MiqAeClassHelper
+  needs :@record
+
+  def role_allows_feature?
+    role_allows?(:feature => 'miq_ae_domain_edit')
+  end
+
+  def calculate_properties
+    super
+    self[:title] = @error_message if @error_message.present?
+  end
+
+  def disabled?
+    disabled = !editable_domain?(@record) && !domains_available_for_copy?
+    @error_message = N_('At least one domain should be enabled & unlocked') if disabled
```

Comment:
remove the `N_` gettext call and use only `_`, also the ampersand in the message can be changed for `'and'`